routes/index.js

```diff
@@ -1,15 +1,32 @@
 const express = require('express');
 const router = express.Router();
 const fs = require('fs');
+const Topic = require('../models/topic')
+const formidable = require('express-formidable');
+
+
+router.use(formidable());
+
+
+router.post('/save-post', (req, res) => {
+ 
+
+    mongoose.connect(mongoConnection);
+    const newPost = new Post(req.fields);
+
+ 
+  
+ });
```

Comment:
Mohammad is working on the logic to add topics. Could we please remove this for now?